src/assets/data/Projects.ts

```diff
@@ -12,7 +12,7 @@ export const projects: ProjectInt[] = [
   {
     title: 'Becca Lyria',
     image: 'project1.png',
-    liveUrl: 'https://beccalyria.nhcarrigan.com',
+    liveUrl: 'https://www.beccalyria.com/',
     codeUrl: 'https://github.com/nhcarrigan/Becca-Lyria',

```

Comment:
```suggestion
    liveUrl: 'https://www.beccalyria.com/discord-documentation',
    codeUrl: 'https://github.com/BeccaLyria/discord-bot',
```

Let's use the path specific for the discord bot site - and update the code URL too :)